portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.servlets;
+
+/*
+Class for a comment to add to the website.
+Contains name and message private fields.
+*/
+public class Comment {
+
+    private String name;
+    private String message;
+
+    public Comment(){
```

Comment:
In creating the Comment class, my intuition was that, since I may want to add more fields to my comments in the future, this would be an extensible approach to do so. That is because this would allow the programmer to easily add new fields to the comments to display. If not storing the comments using a Comment class, I was wondering what you would suggest I do instead (if I am interpreting your suggestion correctly). 

Or, is the suggestion that I should keep the Comment class but change the default values so it isn't no name and no message if the user doesn't input anything for those fields?

Thanks!

Reply:
I think Di's point was that you Comment class is great, but your default constructor is not as useful since we don't really want to post an empty comment (no name or message)

Reply:
Would you recommend changing the default values for the default constructor, or removing the default constructor altogether? Thanks!

Reply:
Is default constructor ever called? Feel free to just remove it if not called.